src/Symfony/Component/Translation/composer.json

```diff
@@ -47,7 +47,8 @@
     "suggest": {
         "symfony/config": "",
         "symfony/yaml": "",
-        "psr/log-implementation": "To use logging capability in translator"
+        "psr/log-implementation": "To use logging capability in translator",
+        "moneyphp/money":  "To use money capability in translator"
```

Comment:
We stopped using `suggest` in the composer packages.

Thus, this suggestion is useless. If you don't already use this money library, there is no reason to suggest you to use it to use the MoneyTranslatable object (you would use its constructor directly). And if you already use the library, composer won't display the suggestion as it is already part of the project.

Reply:
thus, your PR does not even contain the `fromMoney` static method described in the PR description.